server/app/views/components/LinkElement.java

```diff
@@ -84,6 +84,14 @@ public final class LinkElement {
   private boolean doesOpenInNewTab = false;
   private Optional<Icons> icon = Optional.empty();
 
+  public enum IconPosition {
```

Comment:
Given that you added IconPosition - can you update this code to use it as well: https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/app/views/applicant/ProgramIndexView.java?L322 ?